odl-client/src/main/java/org/onap/ccsdk/features/sdnr/wt/odlclient/data/NotificationInputBase.java

```diff
@@ -1,3 +1,10 @@
+/*
+ * Copyright (C) 2020 highstreet technologies GmbH Intellectual Property.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ */
 /*
  * ============LICENSE_START=======================================================
```

Comment:
Note I added a EPL license header before the Alv2 because ODL checkstyle enforces it.
ALv2 is compatible with EPL ( the reverse is not true ) so there is no issue there.
Once built, this code is covered de facto by EPL that it is a (weak-)copyleft license because it imports some ODL dependencies under EPL.